NewDawn/NewDawn/HttpUtil/HttpUtil.swift

```diff
@@ -89,11 +82,18 @@ class HttpUtil{
         task.resume()
     }
     
-    static func getURL(path:String, prod:Bool = CONNECT_TO_PROD) -> URL {
+    static func getURL(path:String, prod:Bool = CONNECT_TO_PROD, isMedia: Bool = false) -> URL {
         var final_path = path
         if final_path.hasPrefix("/") == false {
             final_path = "/" + path
         }
+        if isMedia == true {
```

Comment:
Can you explain more about your usage of "isMedia"?

Reply:
by default, media URLs are not prepended with /api/v1/. So I split them into two different paths

Reply:
I see, but aren't all the current image uploading started with "/api/v1/" ?

Reply:
Image upload is a POST or Multipart API I assume, which starts with /api/v1/
Image download is directly fetching static resources from file system, which is beyond the regular api path: https://github.com/new-dawn/new_dawn_server/blob/master/new_dawn_server/settings.py#L138

Reply:
Got it. Thanks for explanation.